imports/api/coordinator/methods.js

```diff
@@ -116,6 +116,12 @@ function attemptToAddUserToIncident(uid, incidentId){
   var minSituationNeed = null;
 
   incident.situationNeeds.forEach((sn)=>{
+    /** TODO(rlouie):
+     * 1) the situationNeed affordance is still a {String}, but should be written as "description#detectorId"
```

Comment:
this function is going to be replaced -- there shouldn't have to be changes that touch any other than database schema and the matching function from my understanding of everything

Reply:
alright i'll remove the changes to comments in this file so it won't be part of the PR